### DISCUSSION TITLE
Info required: `ros2` driver compatibility with rev 7 sensors

### DISCUSSION THREAD

**Discussion by smkyle90** [ouster-lidar/ouster-ros#122] [Q&A] [answered]
Are the `ros2` drivers compatible with revision 7 OS0 LiDARs? I believe their FW is now 3.X. Current README states "Ouster sensors with FW v2.0 or later". Confirming this includes FW 3.X.

**Comment by Samahu** [comment, marked as answer]
Yes, ros1 and ros2 ouster drivers both are compatible with OS rev7 LiDARs (which require FW 3.0).
